docs/docs/guides/quick-start/add-dependencies.md

```diff
@@ -120,4 +120,8 @@ struct ContentView_Previews: PreviewProvider {
 }
 ```
 
-Run the app from Xcode, and you should see the image loaded from the URL.
\ No newline at end of file
+Run the app from Xcode, and you should see the image loaded from the URL.
+
+## Check your dependencies on implicitness
+
+Run the command `tuist lint implicit-imports` to lint the project for implicit imports. This command will scan all source files and display any implicit imports present in your codebase. To make an import explicit, simply add the imported module to the target in your Tuist project.
```

Comment:
I'd not add this to a quick-start tutorial, which is focused on guiding the person throughout the most important features. What do you think if we add a new page under "Develop", "Validation", where the first page is "Lint".